networking/interconnect/README.md

```diff
@@ -0,0 +1,99 @@
+﻿# Interconnect Attachment and BGP Session
```

Comment:
As mentioned in a PR comment, this will need to be moved under `modules` and renamed.